### PR TITLE
Add crypt module

### PR DESCRIPTION
**Pull request by msabramo**
#367 was too optimistic about `fcntl` and `asyncio`, so I pulled out the addition of `crypt` into a separate PR.

### CODE REVIEW

**Comment by msabramo** [comment]
Is this good to go?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 **User improvements:**
 
-- The built-in `sqlite3** module of Python is now enabled.
+- Support for built-in modules:
+  - `sqlite3`
+  - `bz2`
+  - `mmap`
```
Comment (rth) [inline CHANGELOG.md]: Probably better to put this in the PR that adds support for those (https://github.com/iodide-project/pyodide/pull/364)

Comment (msabramo) [reply CHANGELOG.md]: Thanks! Updated.

**Comment by msabramo** [comment]
@mdboom: Can I ignore the `test-chrome` failure?

**Pull request merged**